static/js/map.js

```diff
@@ -2039,6 +2039,11 @@ function showGymDetails(id) { // eslint-disable-line no-unused-vars
                 '</center>'
         }
 
+        var park = ''
+        if (result['park'] !== 'None' && result['park'] !== undefined) {
+            park = 'Park: ' + result['park']
```

Comment:
Would you be able to wrap Park in the i8ln function to allow for translation. For example:
```javascript
park = i81ln('Park') + ': ' + result['park']
```

Reply:
Just realised this function doesn't currently exist on the `develop` branch so hold fire. Apologies!